cypress/utils/utils.ts

```diff
@@ -1313,7 +1331,7 @@ export function deleteAllBusinessServices() {
     deleteAllRows();
 }
 
-export function deleteAllStakeholderGroups(cancel = false): void {
+export function deleteAllStakeholderGroups(): void {
```

Comment:
It might be used somewhere ? we don't need this change in this PR

Reply:
the parameter itself is not being used in the function, that's why I removed it

`export function deleteAllStakeholderGroups(): void {
    Stakeholdergroups.openList();
    deleteAllRows();
}
`

Reply:
I've re-added it